src/merge.py

```
@@ -83,68 +83,68 @@ def validate_2fa(api: PyiCloudService) -> bool:
             if not result:
-                print_step(TAG_2F_AUTH, "Failed to request trust. You will likely be prompted for confirmation again in the coming weeks", True)
+                print_step(TAG_2F_AUTH, "Failed to request trust. You will likely be prompted for confirmation again in the coming weeks", one_liner=True)
 
     elif api.requires_2sa:
-        print_step(TAG_2F_AUTH, "Two-step authentication required. Your trusted devices are:", True)
+        print_step(TAG_2F_AUTH, "Two-step authentication required. Your trusted devices are:", one_liner=True)
 
         devices = api.trusted_devices
         for i, device in enumerate(devices):
-            print_step(TAG_2F_AUTH, f"  {i}: {device.get('deviceName', 'SMS to %s' % device.get('phoneNumber'))}", True)
+            print_step(TAG_2F_AUTH, f"  {i}: {device.get('deviceName', 'SMS to %s' % device.get('phoneNumber'))}", one_liner=True)
```

Ruff (0.15.7)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)